### DISCUSSION TITLE
-all command option

### DISCUSSION THREAD

**Discussion by molitona** [projectdiscovery/subfinder#638] [Q&A] [answered]
Hi, what does "-all" adds to subfinder ?

If I run a search subfinder won't use all passive sources with/without api keys unless "-all" option is specified  or what ?

see that https://muhdaffa.medium.com/maximizing-the-potential-of-the-subfinder-562fc7e7e9e4

**Comment by ehsandeep** [comment, marked as answer]
@molitona as default, few sources are known to be slow are excluded, and when `-all` flag is used, all the sources are used, you can use `-v` flag to see what sources are default vs with `-all` option.

**Comment by molitona** [comment]
thanks!!
